### DISCUSSION TITLE
Making a new in-game leaderboard option.

### DISCUSSION THREAD

**Discussion by wixxy123** [ppy/osu#23421] [Ideas]
When you are hunting for leaderboard spots, it is almost impossible to tell if you are going to get the spot that you want on the leaderboard mid game. What I mean by this is all you see for the in game board is what your current score is, and then the top 50's score. What if you were able to see your pace compared to other players.
 There is a go osu memory thing that looks like this:
https://www.youtube.com/watch?v=-vi1s49nmSM (skip to around 3:40 to see it at its best.)

You can see that he is only a quarter way through the map but the leaderboard is putting an expected score with a leaderboard spot if he does get that score. Overall, this would really help to snipe people and go for leaderboard spots.

If you do want further examples of this of a way to modify the in-game leaderboard options, look at the steam game Quaver. This thing that I'm talking about came mostly from Quaver, and it is an amazing feature. It would be cool to see it in lazer :). 

I did do a search for other discussions like this. I couldnt find any, but if there is, I am sorry for reposting something that has already been talked about.

**Comment by cdwcgt** [comment]
I feel like it's more of an assumption of the highest score.
Use the highest possible score instead of the current score on the leaderboard.

its seems to be very useful for mania, but other modes that require combos may not be suitable.

but it is a fun idea
